scene/main/window.cpp

```diff
@@ -1101,6 +1101,14 @@ void Window::popup_centered_ratio(float p_ratio) {
 void Window::popup(const Rect2i &p_screen_rect) {
 	emit_signal(SNAME("about_to_popup"));
 
+	if (!_get_embedder()) {
```

Comment:
```suggestion
	if (!_get_embedder() && get_flag(FLAG_POPUP)) {
```

It should be necessary only when opening an unfocusable Popup window, the `popup` method can be used to open other windows as well, and in this case it will send two `NOTIFICATION_WM_WINDOW_FOCUS_OUT`.

Fix seems fine, but I have not fully tested it (seems to work on Linux and macOS, but I was not able to reproduce original issue).

Reply:
Thank you for pointing me to this flag. I updated the PR.